.happy/terraform/modules/ecs-stack/county_runs.tf

```diff
@@ -14,7 +14,7 @@ module nextstrain_scc_local_sfn_config {
   swipe_comms_bucket    = local.swipe_comms_bucket
   swipe_wdl_bucket      = local.swipe_wdl_bucket
   sfn_arn               = module.swipe_sfn_spot.step_function_arn
-  schedule_expressions  = contains(["prod", "staging"], local.deployment_stage) ? ["cron(0 17 ? * 1-5 *)"] : []
+  schedule_expressions  = contains(["prod", "staging"], local.deployment_stage) ? ["cron(0 5 ? * 2-7 *)"] : []
```

Comment:
I believe AWS supports human-readable day-of-week expressions, so instead of 2-7 let's use MON-FRI?
https://docs.aws.amazon.com/AmazonCloudWatch/latest/events/ScheduledEvents.html

Reply:
In the link you sent, 1-7 is SUN-SAT, so I spent quite a while to map the day of week to numbers. 2-7 is Mon-Sat UTC so Sun-Fri PST- but in the end I could have just used day of week lol. Let me change this to de-confuse future ppl. Thanks!!